### PR TITLE
Ariba lablog

### PR DESCRIPTION
**Pull request by GuilleGorines**
Issue #113 should be about done with this PR!

I updated the template except for step 02-srst, that will be taken care of later on.

### CODE REVIEW

File: bu_isciii/templates/services.json [hunk 1]
```diff
@@ -188,5 +186,22 @@
       "last_folder":"REFERENCES",
       "delivery_md": "",
       "delivery_pdf": ""
+    },
+    "ARIBA_characterization": {
```
Comment (saramonzon) [inline bu_isciii/templates/services.json:190]: lower case!

File: bu_isciii/templates/services.json [hunk 2]
```diff
@@ -188,5 +186,22 @@
...
+      "end": "",
+      "description": "",
+      "clean": {
+        "folders":[""],
+        "files":[]
+      },
+      "no_copy": ["RAW", "TMP", "00-reads"],
+      "last_folder":"REFERENCES",
+      "delivery_md": "",
+      "delivery_pdf": ""
```
Comment (saramonzon) [inline bu_isciii/templates/services.json:205]: This should be filled?

**Review by saramonzon — APPROVED** [review]

**Comment by saramonzon** [comment]
lablog files are empty files for ariba!

File: bu_isciii/templates/services.json (hunk 2 above)
Comment (GuilleGorines) [reply bu_isciii/templates/services.json:221]: Honestly I had no idea what to write in here, so I chose the placeholder

**Pull request merged**